syft/execution/translation/torchscript.py

```diff
@@ -17,13 +17,29 @@ def translate(self):
 
         # Temporarily remove reference to original function
         tmp_forward = plan.forward
-        plan.forward = None
 
-        # To avoid storing Plan state tensors in torchscript, they will be send as parameters
+        # To avoid storing Plan state tensors inside the torchscript,
```

Comment:
Seems a bit unrelated to the feature being added in this PR. Is there a separate PR for this fix?